installer/DetailsHandler.php

```diff
@@ -152,7 +151,7 @@ public function getContactNumber()
         $this->contactNumber = $this->contactNumber ? $this->contactNumber : $configurationDataSet["contactNumber"];
 
         if (!$this->isValidUserInput("contact number", $this->contactNumber)) {
-            $this->contactNumber = $this->takeUserInput("Invalid contact number. Please enter contact number:");
+            $this->contactNumber = $this->takeUserInput("Please enter valid contact number (eg:0777888666):");
```

Comment:
why have a sample telephone number? remove this. This can be an individuals actual telephone number. Just the message "Please enter a valid contact number:" is enough.

Reply:
Done